docker-app/qfieldcloud/authentication/tests/test_authentication.py

```diff
@@ -80,6 +82,70 @@ def test_login_logout(self):
         self.assertEqual(len(tokens), 1)
         self.assertLess(tokens[0].expires_at, now())
 
+    def test_login_case_insensitive(self):
+        response = self.login("user1", "abc123")
+        tokens = self.user1.auth_tokens.order_by("-created_at").all()
+
+        self.assertEqual(len(tokens), 1)
+        self.assertTokenMatch(tokens[0], response.json())
+        self.assertGreater(tokens[0].expires_at, now())
+
+        response = self.login("USER1", "abc123")
+        tokens = self.user1.auth_tokens.order_by("-created_at").all()
+
+        self.assertEqual(len(tokens), 2)
+        self.assertTokenMatch(tokens[0], response.json())
+        self.assertGreater(tokens[0].expires_at, now())
+
+    def test_prevent_duplicate_case_insensitive_registration(self):
```

Comment:
```suggestion
    def test_username_uniqueness_is_case_insensitive(self):
```

I would suggest we drop "registration" here, because we don't follow the full process of registering. I am ok if you adjust the suggested name further.

Reply:
Agree!